linux/setup_repo.sh

```diff
@@ -7,6 +7,26 @@ if [[ ${EUID} -ne 0 ]]; then
      exit 1
 fi
 
+# Check for apt version >= 2.4.0 (if found, assuming Debian-based compatible with repo)
```

Comment:
Maybe link the thread to the issue too.

Reply:
@houbsta I meant adding a link to the discussion too. You can still have the launchpad bug here.